spec/index.html

```diff
@@ -820,20 +820,36 @@ <h3><code>rdf:nil</code></h3>
 
 <section id="ch_ttrei">
   <h2>Triple-term based reification</h2>
-  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract 
-    Syntax specification [[RDF12-CONCEPTS]], referring to an RDF triple that 
-    is used as a value, specifically, in the object position of another triple. 
-    Importantly, a triple term does not imply assertion; that is, the triple it 
-    denotes is not necessarily asserted in the RDF graph. This enables the 
-    representation of metadata or commentary about triples that may or may not 
-    be part of the asserted graph content, including potentially contradictory 
-    relationships. For a triple term to be considered an asserted triple, it 
-    must also appear explicitly in an RDF graph as a top-level triple. Otherwise, 
-    it remains unasserted and serves primarily as a reference or subject of metadata.</p>
+  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract Syntax specification 
+    [[RDF12-CONCEPTS]], referring to an RDF triple that is used as a value, specifically, in the object 
+    position of another triple. This use is a reference to a proposition, and importantly, a triple term 
+    does not by itself imply assertion; that is, the triple it denotes is not necessarily asserted in 
+    the RDF graph. For a triple term to also be considered an asserted triple, it must appear explicitly 
+    in the same RDF graph both as an asserted top-level triple and as an unasserted triple term. 
+    Otherwise, it remains unasserted and serves primarily as a reference or subject of metadata. 
+    This enables the representation of metadata or commentary about triples that may or may not be part 
+    of the asserted graph content, including potentially contradictory relationships.</p>
+
+  <p>RDF terms that appear in a triple term have the same denotation as when they appear in an asserted 
+    triple in the graph. For this reason, triple terms are said to be <em>transparent</em>.</p>
+
   <p>A triple term can be the object of a triple whose predicate is 
     <a href="#ch_reifies"><code>rdf:reifies</code></a>. In such cases, the resulting triple is referred 
-    to as a reifying triple, and its subject is termed a reifier. The reifier 
-    can be used to make further statements about the triple term.</p>
+    to as a <em>reifying triple</em>, and its subject is termed a <em>reifier</em>. The reifier can be 
+    used to make further statements about the triple term. When the triple term of a reifying triple 
```

Comment:
The sentence "The reifier can be used to make further statements about the triple term" is quite problematic.
First, it somehow mixes the syntactic level and the semantic level: you don't really make a statement about a term, you make a statement about the thing denoted by the term... We use this kind of metonymy every now and then, and that's generally OK (IMO), but this topic tricky and requires precision.
Second, even if we replace "triple term" with "proposition", then this statement is, at best, confusing: the reifier does not denote the proposition but *something* loosely related to the proposition. Granted, one could say that the statement are *indirectly* about the proposition, but then it should be spelled out.

I propose to simply remove this sentence. The next paragraph makes, IMO, a better job at describing the role of the reifier.
```suggestion
    to as a <em>reifying triple</em>, and its subject is termed a <em>reifier</em>. 
    When the triple term of a reifying triple 
```